### PR TITLE
add profile option

### PR DESCRIPTION
**Pull request by azihsoyn**
Hi! I add `profile` option for (https://github.com/physera/onelogin-aws-cli/issues/16)

I changed alias to profile acording to aws cli userguide. (http://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html)

### CODE REVIEW

File: onelogin_aws_cli/__init__.py [hunk 1]
```diff
@@ -34,9 +34,10 @@ def user_choice(question, options):
 
 
 class OneloginAWS(object):
-    def __init__(self, config):
+    def __init__(self, config, args):
```
Comment (slycoder) [inline onelogin_aws_cli/__init__.py]: Where are args getting passed in?

Comment (azihsoyn) [reply onelogin_aws_cli/__init__.py]: oops! sorry.

Comment (azihsoyn) [reply onelogin_aws_cli/__init__.py]: I fixed it in https://github.com/physera/onelogin-aws-cli/pull/17/commits/d78b4821fd4a778b89af33e6bd4f9eb1ad17f306

**Comment by slycoder** [comment]
Looks good. Thanks!

**Comment by slycoder** [comment]
Looks good. Thanks!

**Comment by azihsoyn** [comment]
Thanks for reviewing and merging!

**Pull request merged**